src/ert/analysis/_es_update.py

```diff
@@ -729,11 +734,12 @@ def _create_smoother_snapshot(
     global_scaling: float,
 ) -> SmootherSnapshot:
     return SmootherSnapshot(
-        prior_name,
-        posterior_name,
-        analysis_config.alpha,
-        analysis_config.std_cutoff,
+        source_case=prior_name,
```

Comment:
Will this end up as "source_case" in `storage`?
If so, perhaps good to use `ensemble` instead of `case` to avoid possible future migrations?

Reply:
Yes it will. I will change to ensemble